Utilities/OTOLocationHelper.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  LocationHelper.swift
+//  OTOnexus
+//
+//  Created by Nicholas Schlueter on 1/10/18.
+//
+
+import Foundation
+import CoreLocation
+
+class OTOLocationHelper: NSObject {
+    static let shared = OTOLocationHelper()
+    fileprivate let locationManager = CLLocationManager()
+    
+    var currentLocation:CLLocation? {
+        return locationManager.location
+    }
+    
+    var userDeniedLocation:Bool {
+        return CLLocationManager.authorizationStatus() == .denied
+    }
+    
+    override init() {
+        super.init()
+        locationManager.desiredAccuracy = kCLLocationAccuracyBest
+        locationManager.delegate = self
+    }
+    
+    func start() {
+        if CLLocationManager.authorizationStatus() == .notDetermined {
+            locationManager.requestWhenInUseAuthorization()
```

Comment:
This will work just fine, but if I understand CLLocationManager correctly, `CLLocationManager.locationServicesEnabled()` will return `false` if the user enabled location services at the initial prompt for the app, but then disables location services globally in Settings.app, in case we want to handle this in the future.

Reply:
I tested a bunch a bunch of different cases and I think it does the right thing in every case I tested.